src/twisted/web/sux.py

```diff
@@ -149,17 +149,19 @@ def dataReceived(self, data):
         stateTable = self._buildStateTable()
         if not self.state:
             # all UTF-16 starts with this string
-            if data.startswith('\xff\xfe'):
-                self._prepend = '\xff\xfe'
+            if data.startswith(b'\xff\xfe'):
```

Comment:
right, would need to also set `self._prepend = data[0:2]` or the like; not necessary, but the branches are the same...